Toolset/home.livecodescript

```diff
@@ -723,12 +672,359 @@ private command revInternal__SetupSplash
+   put "true" into tPropsArray["cREVMacOSMenubar"]
+   put "" into tPropsArray["cREVOpenTabPositions"]
+   put "false" into tPropsArray["cREVProfileReadOnly"]
+   put "Yellow" into tPropsArray["revHelpHiliteColor"]
+   put "true" into tPropsArray["cToolbarText"]
+   put "Pink" into tPropsArray["Pink"]
+   put "true" into tPropsArray["cREVScriptDebugMode"]
+   put "21" into tPropsArray["cLabelFieldHeight"]
+   put "120" into tPropsArray["cREVGraphicWidth"]
+   put "417,99" into tPropsArray["cREVMsgTabRectMultiple Lines"]
+   put "30" into tPropsArray["Num Recent Files"]
+   put "568,259" into tPropsArray["cREVProfileEditorTopLeft"]
+   put "417,113" into tPropsArray["cREVMsgTabRectfrontScripts"]
+   put "1088781994" into tPropsArray["cTipsShown"]
+   put "11" into tPropsArray["cREVRotatePrev3"]
+   put "" into tPropsArray["cREVScriptSize"]
+   put "338" into tPropsArray["cREVButtonSecondTabHeight"]
+   put "true" into tPropsArray["cREVHelpText"]
+   put "160" into tPropsArray["cTabButtonHeight"]
+   put "287,261" into tPropsArray["cREVMsgTabRectrevstack"]
+   put "2" into tPropsArray["cShowTips"]
+   put "false" into tPropsArray["cAppOverview"]
+   put "" into tPropsArray["cCustomerNumber"]
+   put "417,113" into tPropsArray["cREVMsgTabRectGlobal Properties"]
+   put "true" into tPropsArray["cShowAudioclipclips"]
+   put "revolutionli" into tPropsArray["cREVDBName"]
+   put "true" into tPropsArray["cDocsQuickLinksVisible"]
+   put "true" into tPropsArray["cStartPageShow"]
+   put "278" into tPropsArray["cREVErrorsPaletteExpandedHeight"]
+   put "" into tPropsArray["cREVScriptColor"]
+   put "true" into tPropsArray["cShowAudioclips"]
+   put "filtering" into tPropsArray["revHelpListMethod"]
+   set the customProperties[""] of stack "revPreferences" to tPropsArray
+   delete variable tPropsArray
+   
+   -- cREVMessageBoxIndex
+   put "" into tPropsArray["Single Line"]
+   set the customProperties["cREVMessageBoxIndex"] of stack "revPreferences" to tPropsArray
+   delete variable tPropsArray
+   
+   -- cREVMessageBoxNoMessages
+   put "" into tPropsArray["Single Line"]
+   set the customProperties["cREVMessageBoxNoMessages"] of stack "revPreferences" to tPropsArray
+   delete variable tPropsArray
+   
+   -- revHelpWindowPosition
+   set the customProperties["revHelpWindowPosition"] of stack "revPreferences" to tPropsArray
+   delete variable tPropsArray
+   
+   -- cRevDatabaseQuery
+   put "" into tPropsArray["cursorid"]
+   put "" into tPropsArray["connectionid"]
+   set the customProperties["cRevDatabaseQuery"] of stack "revPreferences" to tPropsArray
+   delete variable tPropsArray
+   
+   -- cABDoubleClick
+   put "Play" into tPropsArray["videoclips"]
+   put "Toplevel" into tPropsArray["stacks"]
+   put "Property Inspector" into tPropsArray["controls"]
+   put "Play" into tPropsArray["audioclips"]
+   put "Go" into tPropsArray["cards"]
+   set the customProperties["cABDoubleClick"] of stack "revPreferences" to tPropsArray
```

Comment:
Unfortunately I tried commenting out the AB related stuff and it errors out a couple of lines in when setting the rect to `put "146,123,800,408" into tPropsArray["cAppBrowserRect"]`